tests/path_gateway_dag_test.go

```diff
@@ -541,6 +553,7 @@ func TestNativeDag(t *testing.T) {
 			},
 			{
 				Name: Fmt("HEAD {{name}} with only-if-cached for missing block returns HTTP 412 Precondition Failed", row.Name),
+				Spec: "specs.ipfs.tech/http-gateways/path-gateway/#only-if-cached",
```

Comment:
@laurentsenta  lgtm, but cosmetic nit: dropping `https:` does not buy us much, could we  keep full URLs?
It will enable us to use automated tools that find URLs in strings and report broken ones without extra step, users will be able to click the URL in their code editor etc.

Reply:
Will fix, thanks!